includes/admin/class-wc-payments-admin.php

```diff
@@ -375,7 +375,7 @@ public function add_payments_menu() {
 			]
 		);
 
-		if ( $this->account->is_account_rejected() ) {
+		if ( $this->account->is_account_rejected() || $this->account->is_account_under_review() ) {
```

Comment:
@oaratovskyi I am not sure this is a good behavior for the hard_block scenario. This would mean that merchants are unable to see their deposits, transactions, etc. I think we should let them see those pages. Rejected accounts are different as that is a final state, while hard_block is a more of a "don't allow anything to happen while we look into it".

Reply:
@vladolaru I thought so too! At first I didn't do this check. 
But it behaved really bad. The merchant was able to see inbox notes, action items. But the issue is that they probably shouldn't do anything until we review. Deposits and transactions getting resulted in an error. So I thought that it'll make sense to restrict them rather than fixing all these errors. Now merchants can't do anything if they're suspended by us (hard block) 
I think if we allow them to do actions, there is risk of unknown side-effects. It'll take way more time to fix it and test everything how it works. Even if we do that, we still may get even more support requests when merchants will be suspended and they'll try some actions that they shouldn't.

Those are my thoughts. I think we find it important for merchants, we can create another issue to fully support the account settings, deposits etc for merchants under review, because it increases the scope of the issue.
AFAIK, they spend not many time in hard block state and will be able to get back to green once approved. Otherwise they just remain suspended. WDYT? Does it make sense?

Reply:
Oh! Ok, if that is the case, then yeah, leave it as such. But at least add a comment there to detail this compromise for our future selves.

About creating a separate issue, I think we should probably just chat with Fraud Squad first and get their view on what are their expectations for a hard blocked account.

Reply:
I'll leave a comment, thanks for that!

Reply:
Added the comment in https://github.com/Automattic/woocommerce-payments/pull/8323/commits/9b744270cc7d95089c8665d4c97b78c0a705d9d4